src/test/java/io/openshift/booster/OpenShiftIT.java

```diff
@@ -76,4 +65,84 @@ public void testGreetingEndpointWithNameParameter() {
                 .body("content", is("Hello John from a ConfigMap!"));
     }
 
+    @Test
+    public void testCConfigMapUpdate() {
+        verifyEndpoint();
+        updateConfigMap();
+        rolloutChanges();
+        waitForApp();
+        when().get()
+                .then()
+                .statusCode(200)
+                .body("content", is("Bonjour World from a ConfigMap!"));
+    }
+
+    @Test
+    public void testDConfigMapNotPresent() {
+        get().then()
+                .statusCode(200);
+        assistant.client()
+                .configMaps()
+                .withName(CONFIG_MAP_NAME)
+                .delete();
+        rolloutChanges();
+        await().atMost(5, TimeUnit.MINUTES)
+                .catchUncaughtExceptions()
+                .until(() ->
+                        get().then()
+                                .statusCode(500)
+                );
+    }
+
+    private void verifyEndpoint() {
+        when().get()
+                .then()
+                .statusCode(200)
+                .body("content", is("Hello World from a ConfigMap!"));
+    }
+
+    private void updateConfigMap() {
+        assistant.client()
+                .configMaps()
+                .withName(CONFIG_MAP_NAME)
+                .edit()
+                .addToData("application.properties", "greeting.message: Bonjour %s from a ConfigMap!")
+                .done();
+    }
+
+    private void rolloutChanges() {
+        scale(0, assistant.applicationName());
```

Comment:
I don't like the scale method, it should be a part of assistant as it is used in swarm tests as well. However, moving the code to openshift-test-utils, publishing it, updating all boosters to the new version is something I wanted to avoid:)